src/Features/LanguageServer/Protocol/Handler/Completion/CompletionItemExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Completion;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+{
+
+    internal static class CompletionItemExtensions
+    {
+
+        // A list of the different property names that the insertion text of a completion item may be stored under.
```

Comment:
@heejaechang @dibarbet I'm sure there's a better way to implement this functionality now that this code lives in the Roslyn code base. Let me know!